config/action.d/firewallcmd-common.conf

```diff
@@ -5,72 +5,102 @@
 
 [Init]
 
+# Option:  bantime
+# Notes:   Specifies the bantime in seconds (handled internally rather than by fail2ban)
+# Values:  [ NUM ]  Default: 600
+
+bantime = 600
```

Comment:
Or are we passing bantime setting into the jails actions and I just forgot about that, and it is a non issue? :-)